.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true  # default
+contact_links:
+- name: '💬 IRC: #pypa @ Freenode'
+  url: https://webchat.freenode.net/#pypa
+  about: Chat with devs.
+- name: 🤷💻🤦 Discourse
+  url: https://discuss.python.org/c/packaging
+  about: Please ask typical Q&A here.
```

Comment:
We should give some examples, like "General ideas for Python packaging, questions about structuring projects"

Reply:
Fair enough. Wording improvements are welcome here :)